p4-16/spec/P4-16-spec.mdk

```diff
@@ -4763,7 +4772,7 @@ they do not have to be explicitly initialized to have a legal value.
 
 (Header types with no fields always have a validity bit.)
 
-## Initializing with default values
+## Initializing with default values { #sec-initializing-with-default-values }
```

Comment:
so perhaps this section should also go away. Structure-valued expressions generalize initializers.

Reply:
What about initializing with default values for all of the types that are neither struct nor header, e.g. enums, `bit<W>`, etc.?

Reply:
You're right.